ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/wan_route_servers.schema.yml

```diff
@@ -51,4 +51,4 @@ keys:
                       description: Interface name.
                     ip_address:
```

Comment:
Should we change this key to `public_ip` for consistency? (probably add a deprecation for the testers sake)

Reply:
deprecate or removed immediately?

Reply:
Removed and code updated